src/pages/guides/getting_started/local_development/mcp_server.md

```diff
@@ -0,0 +1,251 @@
+---
+keywords:
+  - Adobe Express
+  - Express Add-on SDK
+  - MCP
+  - Model Context Protocol
+  - Development Tools
+  - TypeScript
+  - API Documentation
+title: Adobe Express Add-on MCP Server
+description: Learn how to set up and use the Adobe Express Add-on MCP Server for enhanced development workflow in compatible editors like Cursor.
+contributors:
+  - https://github.com/hollyschinsky
+---
+
+# Adobe Express Add-on MCP Server
+
+The Adobe Express Add-on MCP Server provides developers with access to comprehensive documentation and TypeScript definitions for Adobe Express add-on development through the Model Context Protocol (MCP). This enables AI-powered development assistance in compatible editors like Cursor.
+
+## What is MCP?
+
+Model Context Protocol (MCP) is an open standard that enables AI assistants to securely access external data sources and tools. In the context of Adobe Express add-on development, the MCP server provides:
```

Comment:
We can perhaps quote them here https://modelcontextprotocol.io/docs/getting-started/intro